deploy/ansible/README.md

```diff
@@ -27,9 +27,9 @@ Then there is a section with variables for this group, in this case we have only
 After `inventory.list` inventory file is configured one could launch the playbook.
 
 ```
-ansible-playbook -i inventory.list main.yml --key-file "~/.ssh/<key>" --extra-vars "docker_password=<nexus_password>" 
+PROFILE=testnet TAG=develop ansible-playbook -i inventory.list main.yml --key-file "~/.ssh/<key>" --extra-vars "docker_password=<nexus_password>" 
```

Comment:
TAG=debug?

Reply:
As soon as this PR is merged, TAG=develop will work